packages/kit/src/views/Send/pages/SendConfirm/TxFeeContainer.tsx

```
@@ -433,39 +437,139 @@
+        (selectedFeeInfo.gas || selectedFeeInfo.gasEIP1559)
+      ) {
+        const swapInfo = unsignedTx.swapInfo;
+        const internalSwapGasLimit = swapInfo.swapBuildResData.result.gasLimit;
+        const internalSwapRoutes = swapInfo.swapBuildResData.result.routesData;
+
+        if (!isNil(internalSwapGasLimit)) {
+          specialGasLimit = internalSwapGasLimit.toString();
+        } else if (internalSwapRoutes && internalSwapRoutes.length > 0) {
+          const allRoutesLength = internalSwapRoutes.reduce(
+            (acc, cur) =>
+              acc.plus(cur.subRoutes?.flatMap((sub) => sub).length ?? 1),
```

Biome

[error] Avoid unnecessary callback in flatMap call.

You can just use flat to flatten the array.
Safe fix: Replace unnecessary flatMap call to flat instead.


(lint/correctness/noFlatMapIdentity)